src/resolve/resolved-dns-rr.h

```diff
@@ -243,6 +251,13 @@ struct DnsResourceRecord {
                         uint8_t matching_type;
                 } tlsa;
 
+                /* https://tools.ietf.org/html/rfc9460 */
+                struct {
+                        uint16_t priority;
+                        char *target_name;
+                        DnsSvcParam *params;
+                } svcb, https;
```

Comment:
Considering the comment about packing, pahole shows:
```
struct {
        uint16_t   priority;             /*  0x40   0x2 */

        /* XXX 6 bytes hole, try to pack */

        char *     target_name;          /*  0x48   0x8 */
        DnsSvcParam * params;            /*  0x50   0x8 */
} svcb;   
```
though, I'm not sure if there's anything to be done about that?